modules/client-configuration/pages/supported-features-sl-micro.adoc

```diff
@@ -0,0 +1,167 @@
+[[supported-features-sl-micro]]
+= Supported {sl-micro} Client Features
+
+
+[IMPORTANT]
+====
+The operating system you run on a client is supported by the organization that supplies the operating system.
+{sl-micro} is supported by {suse}.
+====
+
+[IMPORTANT]
+====
+{sl-micro} is only supported as regular minion ([systemitem]``default`` contact method) for the time being. We are working on managing it as Salt SSH client ([systemitem]``salt-ssh`` contact method), too.
```

Comment:
We should avoid future statements in our technical guide.  As a compromise, I suggest:

```suggestion
{sl-micro} is currently supported as a {salt} client with the [systemitem]``default`` contact method only.
```

Reply:
@keichwa great catch, THANKS.